core/templates/dev/head/pages/state_editor/StateInteractionEditorDirective.js

```diff
@@ -213,8 +215,20 @@ oppia.directive('stateInteractionEditor', [
                       UrlInterpolationService.getInteractionThumbnailImageUrl);
 
                     $scope.INTERACTION_SPECS = INTERACTION_SPECS;
-                    $scope.ALLOWED_INTERACTION_CATEGORIES = (
-                      GLOBALS.ALLOWED_INTERACTION_CATEGORIES);
+
+                    // Checks that the modal is invoked by exploration_editor or
+                    // skill_editor, topic_editor. 
+                    if ($window.location.pathname.match('/create') || 
```

Comment:
Here you can do like this: `var currentUrl = $window.location.pathname.split("/")[1]`. Then compare `currentUrl` with `create`, `skill_editor` etc.

Reply:
Thanks for your clarification! Fixed it just now.